tools/a-star/.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
you are supposed to put .idea/ to ignore the folder

Reply:
Either works, but sure